cmd/integration_test/build/buildspecs/quick-test-eks-a-cli.yml

```diff
@@ -181,6 +181,8 @@ env:
 phases:
   pre_build:
     commands:
+      - pip3 install boto3
+      - until python3 ${CODEBUILD_SRC_DIR}/cmd/integration_test/build/script/check_concurrent_pipelines.py; do echo "Wait for other test pipelines"; sleep 60; done
```

Comment:
Is this wait going to contribute to the overall code-build pipeline time? Curious if this would muddle any metrics we currently collect or report for the quick e2e test duration.

Reply:
Yes, this will reflect in metrics, but I think it's an acceptable drawback since it's better than ruining other e2e test's results

Reply:
True. We could also emit a wait duration metrics here later to distinguish them from the overall time if it becomes a problem later.